DataScience/IntroToPython/IntroToPythonPandas/blog.md

```diff
@@ -199,14 +191,12 @@ data["Mean"] = new_data
 ```
 We create a Series object by adding the two columns from data together. We continue adding the relevant percentages in a similar fashion, and then divide by the number of percentages we are using. Now that we have our master average, there is a new column we can add to our original data set, which we can do so by directly naming a new column for our DataFrame.
```

Comment:
The structure of the sentence can be reversed to say: "Now that we have our master average, we can add a new column to our original data set by directly naming the new column for our DataFrame."